test/unit/test/utils/get-magic-link.test.js

```diff
@@ -0,0 +1,243 @@
+const { when, resetAllWhenMocks } = require('jest-when')
```

Comment:
fine to keep this - but we don't necessarily need a unit tests for a util function used in acceptance tests